### PR TITLE
Update index.rst

### PR DESCRIPTION
**Pull request by MadsKirkFoged**
### Requirements

* Fill out this template to the extent possible so that this PR can be reviewed in a timely manner.
* Replace the bracketed text below with your own.
* All new code requires tests to ensure against regressions.

### Description of the Change

[ *We must be able to understand the design of your change from this description. If we can't get a good idea of what the code will be doing from the description here, the pull request may be closed at the maintainers' discretion. Keep in mind that the maintainer reviewing this PR may not be familiar with or have worked with the wrapper code being implemented, so please walk us through the concepts.* ]

### Benefits

[ *What benefits will be realized by the code change?* ]

### Possible Drawbacks

[ *What are the possible side-effects or negative impacts of the code change?* ]

### Verification Process

[ *What process did you follow to verify that your change has the desired effects?*

- *How did you verify that all new functionality works as expected?*
- *How did you verify that all changed functionality works as expected?*
- *How did you verify that the change has not introduced any regressions?*

*Describe the actions you performed (e.g. text you typed, commands you ran, etc.), and describe the results you observed.*  

*Please attach here any python test sessions or image captures of testing in other wrappers.* ]

### Applicable Issues

[ *Enter any applicable Issues here.  Use ``Closes #????`` if this PR closes an open issue.* ]

### CODE REVIEW

**Comment by ibell** [comment]
Could you possibly provide the URL to your library?  Also highlight that SharpFluids is using CoolProp behind the scenes, if that is in fact true?

**Comment by MadsKirkFoged** [comment]
I have added a short description and a link to its Github

File: Web/coolprop/wrappers/Csharp/index.rst [hunk 1]
```diff
@@ -6,6 +6,32 @@ C# Wrapper
...
+    using UnitsNet;
+
+- Add this example code in your main ::
+
+    Fluid Water = new Fluid(FluidList.Water);
+    Water.UpdatePT(Pressure.FromBars(1.013), Temperature.FromDegreesCelsius(13));
+    Console.WriteLine("Density of water at 13°C: " + Water.RHO);
+    Console.ReadLine();
+
+- Press '5F' or 'Start' to check that it is working
```
Comment (ibell) [inline Web/coolprop/wrappers/Csharp/index.rst:31]: Should this be F5 instead of 5F?

Comment (MadsKirkFoged) [reply Web/coolprop/wrappers/Csharp/index.rst:31]: Yeah you are right it should be 'F5'

Comment (ibell) [reply Web/coolprop/wrappers/Csharp/index.rst:31]: Can you please fix it?

Comment (MadsKirkFoged) [reply Web/coolprop/wrappers/Csharp/index.rst]: Done

**Pull request merged**